schemas/bridging/raiseIntentResponse.schema.json

```
@@ -0,0 +1,30 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://fdc3.finos.org/schemas/next/bridging/raiseIntentResponse.schema.json",
+  "title": "RaiseIntentResponse",
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "bridgeResponse.schema.json"
+    },
+    {
+      "type": "object",
+      "properties": {
+        "type": {
+          "const": "raiseIntentResponse"
+        },
+        "payload": {
+          "type": "object",
+          "properties": {
+            "intentResolution": {
+              "$ref": "../api/intentResolution.schema.json"
+            }
+          },
+          "required": ["intentResolution"],
+          "additionalProperties": false
+        },
+        "meta": true
```

First response is from desktop agent, result response from app
```suggestion      
        "meta": {
          "type": "object",
          "properties": {
            "sources": {
              "type": "array",
              "items": {
                "$ref": "../api/desktopAgentIdentifier.schema.json"
              },
              "errorSources": {
                "type": "array",
                "items": {
                  "$ref": "../api/desktopAgentIdentifier.schema.json"
                }
              }
            }
          }
        }

```